1.0.0/templates/maya-config.yaml

```diff
@@ -50,6 +50,7 @@ data:
     maya.docs.url={{ .Values.server.docsUrl }}
     feature.mdape2e.disable=true
     feature.tour.disable=true
+    feature.mayaonpremregistration.disable=true
```

Comment:
Should the value be  using the values.yaml variable 'featureSubscriptionDisable'

Reply:
no, as is is mayaonlline specific feature, we should not put it in value.yaml

Reply:
i didn't get you, 
are you saying to get the above value `feature.mayaonpremregistration.disable`  from value.yaml?

Reply:
Can you please mention the purpose of the above variable and that of changing featureSubscriptionDisable to false ? I did not understand that part.